host_vars/mongodb.opencraft.hosting/public.yml

```diff
@@ -0,0 +1,11 @@
+mongodb_user_admin_name: "{{ MONGODB_USER_ADMIN_NAME }}"
+mongodb_user_admin_password: "{{ MONGODB_USER_ADMIN_PASSWORD }}"
+
+mongodb_root_admin_name: "{{ MONGODB_ROOT_ADMIN_NAME }}"
+mongodb_root_admin_password: "{{ MONGODB_ROOT_ADMIN_PASSWORD }}"
+
+mongodb_keyfile_content: "{{ MONGODB_KEYFILE_CONTENT }}"
+
+TARSNAP_KEY: "{{ MONGODB_TARSNAP_KEY }}"
+SANITY_CHECK_SNITCH: "{{ MONGODB_SANITY_CHECK_SNITCH }}"
+TARSNAP_BACKUP_SNITCH: "{{ MONGODB_TARSNAP_BACKUP_SNITCH }}"
```

Comment:
@jbzdak Yes, this sounds better, since we wouldn't have any ugly artifacts in the public repo.  I'll try that and see how it goes.